Makefile

```diff
@@ -481,7 +481,7 @@ $(TEST_LOG_DIR):
 #
 .PHONY: test-go
 test-go: ensure-webassets bpf-bytecode roletester rdpclient $(TEST_LOG_DIR) $(RENDER_TESTS)
-test-go: FLAGS ?= '-race'
+test-go: FLAGS ?= -race -timeout=15m
```

Comment:
Might be nice to add a comment as to why the timeout was increased and if it can ever be reduced again if this is an outlier.